modules/proxy.js

```diff
@@ -7,6 +7,7 @@ const absolution = require('absolution');
 const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
 const appData = require('../modules/appData');
+const cheerio = require('cheerio');
```

Comment:
Little thing, but it feels odd to use `cheerio` here when `jsdom` can do the same thing?

Reply:
Done!